src/user/workflow_monitor_debug.rst

```diff
@@ -0,0 +1,250 @@
+(data ids) for this task are released because the previous steps are not
+yet finished and currently this task has no unprocessed inputs. The
+third column shows the task status and number of pseudo inputs (data
+ids) registered for this task. Each data input corresponds to a unique
+job to be submitted in the computation cluster. In this case the task
+unites 1180 jobs. The third column shows the overall completion progress
+(84% or 1001 jobs) and the failure rate (9% or 64 jobs).
+
+Following columns used for the system debug.
+
+Jobs progress
+-------------
+
+Clicking on the task id or its name on the tasks view the detailed
+information is loaded, as shown on the fig. 5:
+
+.. image:: ../_images/Fig5TaskDetail.jpg
+   :width: 5.95313in
+   :height: 4.4446in
+
+Fig 5. Task details
+
+Here one can see several tables, one of the most important is the jobs
+summary. In this table all jobs of the task are counted and grouped by
+their statuses. Since PanDA uses late jobs generation, a job is
+generated only when the next available input is released.
+
+There are two retry filtration modes supported: drop and non drop. They
+could be switched by clicking the correspondent link in the table head.
+The drop mode hides all failed jobs which were successfully retried and
+shows only failures which are hopeless or not yet addressed by the retry
+module. The drop mode is the default one. The non drop mode shows every
+failure regardless if they were retried. It could be directly specified
+in the query URL as follows::
+
+    https://usdf-panda-bigmon.slac.stanford.edu:8443/task/<taskid>/?mode=nodrop
+
+Logs access
+-----------
+
+PanDA monitor provides central access to logs generated by running jobs.
+A log becomes accessible when a job is in the final state - e.g.
+finished or failed. In the IDF deployment every log is transferred to
+the object store and then available for download from there. There are 2
+kinds of job logs available: the Rubin software output and the Pilot log
+which arrange the job run on the computation node.
+
+To access the job log one should load the job details page first. It is
+accessible as::
+
+    https://usdf-panda-bigmon.slac.stanford.edu:8443/job/<jobid>/
+
+The job page could be also navigated starting from the task page::
+
+    task - > list of jobs in particular state -> job
+
+Once a job page has landed a user should click: Logs -> Pilot job
+stderr. This will download the Rubin SW output.
+
+Real-time logs access
+---------------------
+
+The Rubin jobs on the PanDA queues are also provided with
+(near)real-time logging on Google Cloud Logging. Once the jobs have been
+running on the PandDA queues, users can check the json format job logs
+on `the Google Logs Explorer <https://console.cloud.google.com/logs>`__.
+To access it, you need to login with your Google account of
+**lsst.cloud**, and select the project of "**panda-dev**" (the full name
+is panda-dev-1a74).
```

Comment:
Do all campaign team members have an lsst.cloud account?  I'm assuming they and developers are the ones using PanDA right now.  If widening to data-rights users (i.e., why we're testing submission from IDF), will all of them have lsst.cloud accounts?

Reply:
I agree. I am not sure who will have accounts to access this part. I added a bold line to indicate the realtime logs require lsst cloud account to access. Thanks.